service/util/consts.go

```diff
@@ -14,13 +14,13 @@ const (
 	Many              = "many"
 	Relationship      = "relationship"
 	JSON              = "json"
-	Pod               = "Pod"
+	Pod               = "pod"
 	Owner             = "owner"
 	OwnerType         = "ownertype"
 	ResourceVersion   = "resourceversion"
-	Node              = "Node"
+	Node              = "node"
 	Application       = "application"
-	ClusterName       = "ClusterName"
+	ClusterName       = "clustername"
```

Comment:
This is  used to get http header sent by controller, current controller use "ClusterName" so we have to keep it as same as before

Reply:
changed back.